packages/css-framework/src/components/_badge.scss

```diff
@@ -1,28 +1,28 @@
 $badge-border-radius: 3px;
-$badge-font-weight: 700;
+$badge-font-weight: 600;
```

Comment:
Do we have vars for weights?

Reply:
We have font sizes, but not weights. Could potentially add this in the future, but feels like it's a bit overkill, considering the range is only `100 - 900`.